backend/ttrs/serializers.py

```diff
@@ -5,13 +5,42 @@
 from rest_framework.exceptions import ValidationError
 from django.core.exceptions import ValidationError as DjangoValidationError
 
-from .models import Student, College, Department, Major, Course, Lecture
+from .models import Student, College, Department, Major, Course, Lecture, TimeTable
 
 
 class StudentSerializer(serializers.ModelSerializer):
+    my_time_table = serializers.SerializerMethodField()
+    bookmarked_time_tables = serializers.SerializerMethodField()
+    received_time_tables = serializers.SerializerMethodField()
+
+    def get_my_time_table(self, student):
+        my_table = []
```

Comment:
I've done it because
1. provide some unity with other _getter functions_
2. I wasn't sure whether I should collect my timetables from _previous semesters_ as well or not

Do you think it would be better if my_table contains just one tt?

Reply:
I'm not sure for that, too.
It would better to discuss with our teammates about this problem; whether make multiple 'my time table's for each semesters or make only on 'my time table' regardless of the semester.

Reply:
Yeah. Let's discuss it tomorrow.